client.go

```diff
@@ -110,6 +110,61 @@ func chopContentType(contentType string) string {
 	return strings.Split(contentType, ";")[0]
 }
 
+func (lc *StreamClient) Subscribe(ctx context.Context, group string, f func(ctx context.Context, payload []byte, contentType string) error) error {
+	request := liiklus.SubscribeRequest{
+		Topic:                lc.TopicName,
+		Group:                group,
+		AutoOffsetReset:      liiklus.SubscribeRequest_EARLIEST,
+	}
+	subscribedClient, err := lc.client.Subscribe(ctx, &request)
+	if err != nil {
+		return err
+	}
+
+	subscribeReply, err := subscribedClient.Recv()
+	if err != nil {
+		return err
+	}
+
+	receiveRequest := liiklus.ReceiveRequest{
+		Assignment:           subscribeReply.GetAssignment(),
+		LastKnownOffset:      0,
+	}
+	receiveClient, err := lc.client.Receive(context.Background(), &receiveRequest)
+	if err != nil {
+		return err
+	}
+	for true {
```

Comment:
I think this should happen in a separate goroutine

Reply:
done!